src/main/java/com/gooddata/executeafm/afm/PopMeasureDefinition.java

```diff
@@ -5,21 +5,23 @@
  */
 package com.gooddata.executeafm.afm;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonRootName;
 import com.gooddata.executeafm.ObjQualifier;
 import com.gooddata.executeafm.UriObjQualifier;
 import com.gooddata.util.GoodDataToStringBuilder;
 
 import java.io.Serializable;
-import java.util.Objects;
 
 import static com.gooddata.executeafm.afm.PopMeasureDefinition.NAME;
 
 /**
  * Definition of so called "period over period" measure
  */
 @JsonRootName(NAME)
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Yep, the `offset` field will not be included in the produced JSON when it is null. However, you are right that it affects existing fields in the class, other than the `offset`. Should I rather move the annotation to the `offset` field only?

Reply:
@jimirocks I moved the annotation to the member level to affect only the newly added `offset` field.

Reply:
mea culpa - not only I somehow thought you are removing instead of adding, but also the specification already defines the other fields as nonnull. So the compatibility is not an oissues here.

Reply:
@jimirocks the annotation is back on the class level based on the latest discussion (the REST API specification in the `obj.res` file.